app/controllers/api/v1/workflows_controller.rb

```diff
@@ -82,13 +82,13 @@ def field_context
   # be easily located in specific controller actions
   def post_link_actions(workflow)
     if workflow.set_member_subjects.exists?
+      RefreshWorkflowStatusWorker.perform_async(workflow.id)
       case relation
-      when :retired_subjects, 'retired_subjects'
+      when /\Aretired_subjects\z/
```

Comment:
Not sure how I feel about this. Wouldn't comparing with a regex be slower? What do you think about changing the `case` to be `case relation.to_s`

Reply:
Probably though benchmarks show it is negligible. I assumed reading the one regex is a little easier but the string comparison makes sense too.
```
       user     system      total        real
regex:   0.000000   0.000000   0.000000 (  0.000034)
string:  0.000000   0.000000   0.000000 (  0.000015)
       user     system      total        real
regex:   0.000000   0.000000   0.000000 (  0.000014)
string:  0.000000   0.000000   0.000000 (  0.000015)
       user     system      total        real
regex:   0.000000   0.000000   0.000000 (  0.000011)
string:  0.000000   0.000000   0.000000 (  0.000015)
.       user     system      total        real
regex:   0.000000   0.000000   0.000000 (  0.000025)
string:  0.000000   0.000000   0.000000 (  0.000009)
       user     system      total        real
regex:   0.000000   0.000000   0.000000 (  0.000013)
string:  0.000000   0.000000   0.000000 (  0.000009)
       user     system      total        real
regex:   0.000000   0.000000   0.000000 (  0.000011)
string:  0.000000   0.000000   0.000000 (  0.000011)
```

Reply:
here was the code
``` ruby
3.times do
  n = 10000
  Benchmark.bm do |x|
    x.report("regex: ") do
      case relation
      when /\Asubject_sets\z/ then true
      end
    end
    x.report("string:") do
      case relation.to_s
      when "subject_sets" then true
      end
    end
  end
end
```